string room_id
bool state
---
bool success
string message
